testing/py-tox/APKBUILD

```diff
@@ -0,0 +1,56 @@
+# Contributor: Stuart Cardall <developer@it-offshore.co.uk>
+# Maintainer: Stuart Cardall <developer@it-offshore.co.uk>
+pkgname=py-tox
+_pkgname=${pkgname#py-*}
+pkgver=2.7.0
+pkgrel=0
+pkgdesc="virtualenv management and test command line tool"
+url="https://tox.readthedocs.org/"
+arch="noarch"
+license="MIT"
+_depends="py-virtualenv"
+depends="py-py py-pluggy py-argparse"
+makedepends="python2-dev python3-dev py-setuptools"
+options="!check" # temp disable as requires itself
```

Comment:
the `tox` binary does not yet exist when `check()` runs:

```
>>> py-tox: Entering fakeroot...
/usr/bin/abuild: line 2356: tox: not found
>>> ERROR: py-tox*: check failed
>>> ERROR: py-tox: all failed
>>> py-tox: Uninstalling dependencies...
musl64 [~/aports/testing/py-tox]$ find pkg -type f -name tox
find: ‘pkg’: No such file or directory
musl64 [~/aports/testing/py-tox]$ find src -type f -name tox
musl64 [~/aports/testing/py-tox]$ 
```

the tests need to be enabled after `py-tox` is in the repos.